packages/truffle-core/lib/commands/develop.js

```diff
@@ -38,26 +33,43 @@ var command = {
     Environment.develop(config, testrpcOptions, function(err) {
       if (err) return done(err);
 
-      var c = new Console(console_commands, config.with({
-        noAliases: true
-      }));
+      var c = new Console(
+        console_commands,
+        config.with({
+          noAliases: true
+        })
+      );
 
       c.start(done);
       c.on("exit", function() {
         process.exit();
       });
     });
   },
-  run: function (options, done) {
+  run: function(options, done) {
```

Comment:
Random note (not worth fixing now), but I believe most JS style-guides suggest `function (` instead of no-space.